app/presenters/publishing_api_presenters/detailed_guide.rb

```diff
@@ -83,4 +84,26 @@ def unpublishing_explanation
       Whitehall::GovspeakRenderer.new.govspeak_to_html(item.unpublishing.explanation)
     end
   end
+
+  def national_applicability
+    all_nations = %w(England Northern\ Ireland Scotland Wales)
+    nations = all_nations.reduce({}) { |hash, nation|
+      key = nation.tr(' ', '_').downcase.to_sym
+      hash[key] = {
+        label: nation,
+        applicable: true
+      }
+      hash
+    }
+
+    inapplicabilities = item.nation_inapplicabilities
+    nations = inapplicabilities.reduce(nations) { |hash, inapplicability|
+      key = inapplicability.nation.name.tr(' ', '_').downcase.to_sym
```

Comment:
The common `.tr(' ', '_').downcase.to_sym` between this and above could be split into a separate method too.